internal/controller/helmrepository_controller_test.go

```diff
@@ -550,6 +550,38 @@ func TestHelmRepositoryReconciler_reconcileSource(t *testing.T) {
 				t.Expect(artifact.Revision).ToNot(BeEmpty())
 			},
 		},
+		{
+			// Regression test for: https://github.com/fluxcd/source-controller/issues/1218
+			name:     "HTTP with docker config secretRef makes ArtifactOutdated=true",
```

Comment:
Shouldn't this rather test that the artifact is successfully generated? And I think that's what it actually tests by setting `want: sreconcile.ResultSuccess,` so maybe it's just the test name that's confusing here.

Reply:
this is copied from all the above test cases, with the test condition  bit changed.

Reply:
I don't think it hurts to give this test a more comprehensible name.